.babelrc

```diff
@@ -1,3 +1,27 @@
 {
-  "presets": ["@babel/preset-env"]
+  "presets": [
+    "@babel/preset-typescript",
+    [
+      "@babel/preset-react",
+      {
+        "runtime": "automatic",
+        "importSource": "preact"
+      }
+    ],
+    "@babel/preset-env"
+  ],
+  "env": {
+    "development": {
+      "presets": [
+        [
+          "@babel/preset-react",
+          {
+            "development": true,
+            "runtime": "automatic",
+            "importSource": "preact"
+          }
+        ]
+      ]
+    }
+  }
```

Comment:
I'm fine if you want to make that a separate PR, although I think this is a pretty safe option to add as we've been using it a long time. The documentation mentions that it will become the default option in future Babel releases, so we probably ought to be using it everywhere.

Related to this, we need to get our browserslist updated everywhere to something a bit more modern.

Reply:
Done: https://github.com/hypothesis/h/pull/8789/commits/c204e69c7a5406d96ad37adc0bb169673b137fef